### PR TITLE
SAS vs R Template

### PR DESCRIPTION
**Pull request by orladoylenvs**

### CODE REVIEW

**Review by statasaurus — APPROVED** [review]
This is amazing thank you for adding

File: templates/RvsSAS_template.qmd [hunk 1]
```diff
@@ -0,0 +1,196 @@
+---
+title: "R vs SAS [Name of Method]"
+---
+
+------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (zzzzqqqa) [reply templates/RvsSAS_template.qmd:5]: Sure.

File: templates/RvsSAS_template.qmd [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+# R vs SAS \[Name of Method\]
```
[the remark this answers is not in the document]
Comment (zzzzqqqa) [reply templates/RvsSAS_template.qmd:7]: Thank you for the suggestion. I believe that Markdown documents will eventually be rendered into HTML, and the title metadata can assist in generating the browser tab title and bookmark. Given that this is also included in the R template, I would prefer to keep the title line, if that's alright with you.

File: templates/RvsSAS_template.qmd [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+# R code for basic Poisson Regression
+example_model <- glm(count ~ predictor, family = poisson(link = "log"), data = example_data)
+
+# Summary of the model
+summary(example_model)
+```
+
+#### Results Comparison
+
+Provide a detailed comparison of the results obtained from both SAS and R. Highlight any differences and provide explanations if possible.
```
[the remark this answers is not in the document]
Comment (zzzzqqqa) [reply templates/RvsSAS_template.qmd:104]: Do you mean the option used in the code? How about adding below after each R or SAS code example:
`Provide a detailed description of the scenario, including options utilized in the code, along with a screenshot showcasing the output.`

**Comment by zzzzqqqa** [comment]
> I had added a few suggested changes - apologies they were sitting as pending. Let me know if you'd like us to make these updates.

Hi @orladoylenvs , thank you for the suggestions. Could you please help review the update? https://github.com/PSIAIMS/CAMIS/compare/main...zzzzqqqa:CAMISfork:main

**Comment by statasaurus** [comment]
I think you will need to make another pull request cause I pull this in prior to these changes

**Pull request merged**